src/Servers/Kestrel/Transport.Quic/test/QuicConnectionContextTests.cs

```diff
@@ -421,6 +421,8 @@ public async Task StreamPool_StreamAbortedOnClient_NotPooled()
     public async Task StreamPool_StreamAbortedOnClientAndServer_NotPooled()
     {
         // Arrange
+        using var httpEventSource = new HttpEventSourceListener(LoggerFactory);
```

Comment:
unreferenced❔

Reply:
Correct. It registers an event source in the constructor and removes it on dispose. Since no one uses it, it could be changed to:

```cs
using var _ = new HttpEventSourceListener(LoggerFactory);
```

But I'd prefer to do that in a separate PR to remove them all at once. Personally, I'm happy leaving it as is.